_posts/blog/2016-05-04-new-website.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "New Website"
+headline: "The new Ehcache website is LIVE! Checkout its first post..."
```

Comment:
It's just for tests - What I would suggest is to set hidden: true. It won't appear in the website and not in the RSS feed, and it would still be there as an example.